### PR TITLE
NFT instance token transfers tab

### PR DESCRIPTION
**Pull request by buberdds**
This was built early Dec but it seems Ok using staging API.

sample view request url https://nexus.stg.oasis.io/v1/emerald/events?offset=0&limit=10&type=evm.log&evm_log_signature=ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef&contract_address=oasis1qplvh8ejuqmeyma9a96ycws074wrpt8uty2d2mjj&nft_id=1

### CODE REVIEW

File: src/app/pages/TokenDashboardPage/hook.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export const useTokenInfo = (scope: SearchScope, address: string, enabled = true
   }
 }
 
-export const useTokenTransfers = (scope: SearchScope, address: string) => {
+export const useTokenTransfers = (scope: SearchScope, address: string, nftId?: string) => {
```
Comment (buberdds) [inline src/app/pages/TokenDashboardPage/hook.ts:38]: Do we prefer to keep useTokenTransfers intact and add `useInstanceTokenTransfers` or we are fine with ternary operators?

Comment (lukaw3d) [reply src/app/pages/TokenDashboardPage/hook.ts]: I think a good function signature would be
```ts
export const useTokenTransfers = (scope: SearchScope, params: { account: string } | { nft_id: string, contract_address: string }) => {
```

Comment (lukaw3d) [reply src/app/pages/TokenDashboardPage/hook.ts]: hm, nope, that apparently permits `useTokenTransfers(scope, { account: '4', contract_address: 'a' })`

Comment (buberdds) [reply src/app/pages/TokenDashboardPage/hook.ts]: updated

File: src/app/pages/TokenDashboardPage/hook.ts [hunk 2]
```diff
@@ -46,21 +61,19 @@ export const useTokenTransfers = (scope: SearchScope, address: string) => {
...
-      type: 'evm.log',
+      type: RuntimeEventType.evmlog,
       // The following is the hex-encoded signature for Transfer(address,address,uint256)
       evm_log_signature: 'ddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
+      ...params,
     },
     {
       query: {
-        enabled: !!oasisAddress,
+        enabled: !!params.rel || (!!params.contract_address && !!params.nft_id),
```
Comment (lukaw3d) [inline src/app/pages/TokenDashboardPage/hook.ts:76]: oh :/

Comment (buberdds) [reply src/app/pages/TokenDashboardPage/hook.ts:76]: yes, we are enabling queries when oasis address is returned by `useTransformToOasisAddress`. How do you want to change it?

**Review by lukaw3d — APPROVED** [review]

**Pull request merged**